Sources/XcodeGenKit/SourceGenerator.swift

```diff
@@ -115,31 +115,7 @@ class SourceGenerator {
             }
             let lastKnownFileType = lastKnownFileType ?? PBXFileReference.fileType(path: path)
 
-            if path.extension == "xcdatamodeld" {
-                let models = (try? path.children()) ?? []
-                let modelFileReference = models
-                    .filter { $0.extension == "xcdatamodel" }
-                    .sorted { $0.string.localizedStandardCompare($1.string) == .orderedAscending }
-                    .map { path in
-                        createObject(
-                            id: path.byRemovingBase(path: project.basePath).string,
-                            PBXFileReference(
-                                sourceTree: .group,
-                                lastKnownFileType: "wrapper.xcdatamodel",
-                                path: path.lastComponent
-                            )
-                        )
-                    }
-                let versionGroup = addObject(id: fileReferencePath.string, XCVersionGroup(
-                    currentVersion: modelFileReference.last?.reference,
-                    path: fileReferencePath.string,
-                    sourceTree: sourceTree,
-                    versionGroupType: "wrapper.xcdatamodel",
-                    children: modelFileReference.map { $0.reference }
-                ))
-                fileReferencesByPath[fileReferenceKey] = versionGroup
-                return versionGroup
-            } else {
+            guard path.extension == "xcdatamodeld" else {
```

Comment:
Can we keep the `if` as opposed to the `guard`? Not being a `xcdatamodeld` is not a failing case, and we may handle other filetypes differently in the future as well. 
Additionally it will make these diffs easier to read.